.eslintrc

```diff
@@ -72,7 +72,7 @@
     "babel/generator-star-spacing": ["error", { before: false, after: true }],
 
     "react/display-name": ["error", { ignoreTranspilerName: true }],
-
+    "react/prefer-stateless-function": ["off"],
```

Comment:
Disabling it would stop telling you to turn this
```js
class Foo extends Component {
  render() { return <div/> }
}
```
into a stateless function component. That means we can't easily catch class components that don't have a `shouldComponentUpdate` just by linting. It's definitely not the way this rule was intended to be used though, so I guess I agree with removing it. I'd just like to have something that warns us when we have a class component that doesn't have a `shouldComponentUpdate`, no matter what warning that is.